Framework/Algorithms/src/DeleteWorkspaces.cpp

```diff
@@ -7,6 +7,7 @@
 #include "MantidAlgorithms/DeleteWorkspaces.h"
 #include "MantidAPI/ADSValidator.h"
 #include "MantidAPI/AnalysisDataService.h"
+#include "MantidAPI/WorkspaceGroup.h"
```

Comment:
Could we remove this unused include? Accumulating these plagues our build times...